examples/python-stack/README.md

```diff
@@ -0,0 +1,19 @@
+# Datadog CDK Python Example
+
+Use this example Python stack to try out the [datadog-cdk-constructs](https://github.com/DataDog/datadog-cdk-constructs) v2 library. It contains Node, Python and Go Lambda functions.
+
+## Getting Started
+
+1. Get a Datadog API key to be used to send monitoring data ([Datadog API keys documentation](https://docs.datadoghq.com/account_management/api-app-keys/#add-an-api-key-or-client-token)).
+1. Run `export DD_API_KEY=DATADOG_API_KEY_HERE` to set the Datadog API key in your shell session.
+1. Run `virtualenv env` to create a virtual environment.
+1. Run `source env/bin/activate` to activate the virtual environment.
+1. Run `pip install -r requirements` to install dependencies.
+1. Update the layer versions in `lib/cdk-python-stack.ts` with the latest releases:
+   - Datadog Lambda Extension: https://github.com/DataDog/datadog-lambda-extension/releases
+   - Python Layer: https://github.com/DataDog/datadog-lambda-python/releases
+   - Node Layer: https://github.com/DataDog/datadog-lambda-js/releases
+1. Run `cdk synth` to synthesize the CloudFormation template.
+1. Run `cdk diff` to see the resource and permission changes that are made.
+1. Run `cdk deploy` to deploy the stack to AWS.
+1. Invoke your Lambda functions and look for them in Datadog Serverless Monitoring.
```

Comment:
Maybe we could add an image to the docs of what to expect? And maybe a link to the CDK configuration again?

Reply:
Added screenshots of a Lambda function trace in each readme, links directly to the Serverless page in Datadog for Lambda functions, and a permalink to the CDK configuration in each stack